governance.md

```diff
@@ -117,6 +119,13 @@ that organization. The SC may do some basic checks to ensure that
 candidates are familiar with the project and represent relevant contributors.
 Expansion should be approved by at least 2/3rds of current SC members.
 
+###### Adding a new member of an organization that is already represented in the SC
+
+As the organizations in the project are the ones responsible for selecting the
+individual(s) to represent them, any organization that is already part of the
+project can add a new member at its own will, as long as there are no more than
+two members of the same organization as part of the Steering Committee.
```

Comment:
and something of a quorum vote?

Reply:
I suppose that is fine for replacement. My question was regarding adding additional folks. Whether from an existing company, or a new one. This isn't [directly] a pay-to-play community, despite many of us are paid to work here and that employers may be a CNCF member.